src/plugins/git/README.md

```diff
@@ -0,0 +1,16 @@
+# GIT plugin
+
+## Goal
+The cmd plugin offers user to download code from git repo.
+
+## Schema
+```yaml
+extras:
+  com.microsoft.pai.runtimeplugin:
+    - plugin: git
+      parameters:
+        repo_uri: <git repo>
+        options:
+        - <git clone options>
+        clone_dir: <clone dir>
+```
```

Comment:
Also, what will happen if the default dir or `clone_dir` exists before git cloning?

Reply:
If it has file in same name. will overwrite it. Will doc this. Customer should be care of this case

Reply:
Can we throw an error if this happens? I think overwriting user's data may not be a good behavior.

Reply:
Added doc, thx.

Reply:
> Can we throw an error if this happens? I think overwriting user's data may not be a good behavior.

I think this parameter is set by user, user should aware this. It's user responsibility to take care of this parameter

Reply:
Change to fail the command and throw error. Which respect git default behavior